reana_client/cli/workflow.py

```diff
@@ -477,9 +482,64 @@ def workflow_validate(ctx, file):
             err=True)
 
 
+@click.command(
+    'run',
+    help='Create, upload and start the REANA workflow.')
+@click.option(
+    '-f',
+    '--file',
+    type=click.Path(exists=True, resolve_path=True),
+    default=reana_yaml_default_file_path,
+    help='REANA specifications file describing the workflow and '
+         'context which REANA should execute.')
+@click.argument(
+    'filenames',
+    metavar='SOURCES',
+    type=click.Path(exists=True, resolve_path=True),
+    nargs=-1)
+@click.option(
+    '-n',
+    '--name',
+    default='',
+    help='Name of the workflow.')
+@click.option(
+    '--skip-validation',
+    is_flag=True,
+    help="If set, specifications file is not validated before "
+         "submitting it's contents to REANA server.")
+@click.option(
+    '-p', '--parameter',
+    multiple=True,
+    help='Optional operational parameters for the workflow execution. '
+         'E.g. CACHE=off.',
+)
+@add_access_token_options
+@click.pass_context
+def workflow_run(ctx, file, filenames, name, skip_validation,
+                 access_token, parameter):
+    """Wrapper command for create, upload and start a workflow."""
+    # set context parameters for subcommand
+    ctx.invoked_by_subcommand = True
+    ctx.workflow_name = ""
+    ctx.invoke(workflow_create,
+               file=file,
+               name=name,
+               skip_validation=skip_validation,
+               access_token=access_token)
+    ctx.invoke(upload_files,
+               workflow=ctx.workflow_name,
+               filenames=filenames,
+               access_token=access_token)
+    ctx.invoke(workflow_start,
+               workflow=ctx.workflow_name,
+               access_token=access_token,
+               parameter=parameter)
```

Comment:
Currently the output is:

```console
$ reana-client run -n my-test-analysis
my-test-analysis.3
File code/helloworld.py was successfully uploaded.
File data/names.txt was successfully uploaded.
my-test-analysis.3 has been started.
```

Perhaps we should display leading messages like:

```

```
$ reana-client run -n my-test-analysis
my-test-analysis.3
File code/helloworld.py was successfully uploaded.
File data/names.txt was successfully uploaded.
my-test-analysis.3 has been started.
```

Perhaps we should display leading messages like:

```console
$ reana-client run -n my-test-analysis
>>> Creating workflow... 
my-test-analysis.3
>>> Uploading files... 
File code/helloworld.py was successfully uploaded.
File data/names.txt was successfully uploaded.
>>> Starting workflow...
my-test-analysis.3 has been started.
```

so that users know what's going on.

See also what `reana-dev` does in displaying `reana-client` commands it invokes; that's another possibility (that also "educates" users while they observe the output.)

In any case we should think of decorating (`>>>`) or otherwise colorising the leading messages in similar manner as for other `reana-client` commands.